spotseeker_server/models.py

```diff
@@ -211,6 +210,12 @@ def get_with_external(spot_id):
         else:
             return Spot.objects.get(pk=spot_id)
 
+    def delete(self):
```

Comment:
Once you get to multiple servers, you'd need to move to something like memcached.  Or, put a lifetime on the cache entry and be OK with it if something is displayed that's an hour out of date.